test-conclave-init.sh

```diff
@@ -13,6 +15,12 @@ pushd build
 
 conclaveInitJar=$(find repo/com/r3/conclave/conclave-init/ -name 'conclave-init-*jar' -not -name 'conclave-init-*javadoc.jar' -not -name 'conclave-init-*-sources.jar')
 
+conclaveInitVersion=$("$JAVA_HOME"/bin/java -jar "$conclaveInitJar" -V | awk '{print $3}')
```

Comment:
Sounds good here.
Note a thing though: while testing, I noticed that the previous line does not seem to work when there are multiple jar present. This can happen when we build the SDK multiple times and we have more versions of the jars produced.
Not related to this, might be relevant for a future fix.

Reply:
Okay, let me check if I can fix it quickly or we will leave it for later.

Reply:
Fixed. Now it only picks up the latest created/modified .jar file.